tests/service/PatternMatchCompilationTests.fs

```diff
@@ -249,6 +249,22 @@ match None with
     dumpDiagnostics checkResults |> shouldEqual [
     ]
 
+[<Test>]
+#if !NETCOREAPP
+[<Ignore("These tests weren't running on desktop and this test fails")>]
+#endif
+let ``Union case 10 - incomplete union fields`` () =
+    let _, checkResults = getParseAndCheckResults """
+type U =
+    | B of  f1:int seq * {|X:string|} * f3:U * f4: (int * System.String)
+
+let x : U = failwith ""
+match x with
+| B  -> 42
+"""
+    dumpDiagnostics checkResults |> shouldEqual [
+        "(7,2--7,3): This union case expects 4 arguments in tupled form, but was given 0. The missing field arguments may be any of: f1: seq<int> {| X: string |} f3: U f4: (int * System.String)"
+    ]
```

Comment:
We should be having new tests in the `ComponentTests` suite instead, if they're generic enough (i.e. not service-, autocomplete- or tooling-related).

Reply:
Ok, will see how this can be migrated